kolena/workflow/model.py

```diff
@@ -114,7 +96,20 @@ def __init__(
         except NotFoundError:
             loaded = self.create(name, infer, metadata, tags)
 
-        self._populate_from_other(loaded)
+        with self._unfrozen():
+            self._id = loaded._id
+            self.name = loaded.name
+            """Unique name of the model."""
+            self.metadata = loaded.metadata
+            """Unstructured metadata associated with the model."""
+            self.tags = loaded.tags
+            """Tags associated with this model."""
+            self.infer = loaded.infer
+            """
+            Function transforming a [`TestSample`][kolena.workflow.TestSample] for a workflow into an
+            [`Inference`][kolena.workflow.Inference] object. Required when using [`test`][kolena.workflow.test] or
+            [`TestRun.run`][kolena.workflow.TestRun.run].
+            """
```

Comment:
Example of undesirable docs diff here — type hint is missing, and `loaded.<x>` is noisy. Both are correctly labeled as instance attributes:

| Trunk | Branch |
| --- | --- |
| ![Screenshot 2023-11-27 at 12 19 13 PM](https://github.com/kolenaIO/kolena/assets/9648886/605b821d-1aa9-4eb5-8972-1e4145e8285b) |  ![Screenshot 2023-11-27 at 12 19 20 PM](https://github.com/kolenaIO/kolena/assets/9648886/dbf453d5-a691-4e82-b79a-2126935761d9) |

Reply:
Ah good catch. I missed this when I took a look at the documentation.
Seems to be a limitation for mkdocstrings Sphinx support https://mkdocstrings.github.io/griffe/docstrings/#sections.

Adding `__init__` params as shown on https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#an-example-class-with-docstrings renders as
![Screenshot 2023-11-28 at 4 19 39 PM](https://github.com/kolenaIO/kolena/assets/25123885/4bcf97d8-930f-4909-aa6a-9285afa0949d)

I think we should backlog moving towards google or numpy style comments since mkdocstrings support is lacking for Sphinx.

Reply:
We just migrated away from Sphinx to MkDocs! There are tradeoffs with all of these systems — I don't think we're going to move back any time soon 🙂

Reply:
I'm referring to mkdocstring `docstring_style` which [is set to sphinx](https://github.com/kolenaIO/kolena/blob/7dce11f41a3327565616360ee773210359b20d39/mkdocs.yml#L191), but is poorly supported https://mkdocstrings.github.io/griffe/docstrings/#sections